src/lib/modules/deployment/contract_deployer.js

```diff
@@ -90,10 +93,17 @@ class ContractDeployer {
     }
 
     async.waterfall([
-      function requestBlockchainConnector(callback) {
+      function checkContractBytesize(next) {
+        if(contract.code.length <= MAX_CONTRACT_BYTECODE_LENGTH) return next();
```

Comment:
Very picky, but could you do the opposite?
So do:
```
 if(contract.code.length > MAX_CONTRACT_BYTECODE_LENGTH) {
  return next(new Error(`Bytecode for ${contract.className} contract is too large. Not deploying.`));
}
next();
```
It's less confusing, because we usually break the flow for exceptions.

Reply:
Absolutely!